tests/cgroupfs_label/test

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/perl
+
+use Test;
+BEGIN { plan tests => 2 }
+
+my $ret;
+
+# Create a new cgroupfs directory and relabel it.
+mkdir("/sys/fs/cgroup/unified/test");
+system("chcon -R -t test_cgroup_t /sys/fs/cgroup/unified/test");
```

Comment:
Do we really need -R above?

Reply:
Not really, but it seemed unnatural to relabel just the directory. I don't mind dropping the -R if I have to.